packages/cli/src/CLIApp.tsx

```diff
@@ -28,6 +30,7 @@ export const CLIApp = () => {
     rawData,
   })
 
+  const jarExists = checkForJar()
```

Comment:
This is currently not being used; it looks like you updated the code at some point to instead do this check in `useMenuOptions.tsx`

I think you can remove this (but correct me if I am wrong!)